### PR TITLE
Add case format large size disk based on pass-through block device

### PR DESCRIPTION
**Pull request by qingwangrh**
ID:2066627
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=block_iscsi_format_large_size_disk --guestname=Win2019 --clone=no
/ (1/3) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_virtio.q35: PASS (115.45 s)
 (2/3) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_hd.q35: STARTED
\ (2/3) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_hd.q35: PASS (161.28 s)
 (3/3) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_block.q35: STARTED
| (3/3) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_block.q35: PASS (112.60 s)

**Comment by peixiu** [comment]
Looks good to me, thank you~

python ConfigTest.py --testcase=block_iscsi_format_large_size_disk --guestname=Win2022 --clone=no --firmware=ovmf

 (1/3) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_virtio.q35: STARTED
/ (1/3) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_virtio.q35: PASS (263.47 s)
 (2/3) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_hd.q35: STARTED
/ (2/3) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_hd.q35: PASS (223.33 s)
 (3/3) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_block.q35: STARTED
/ (3/3) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_block.q35: PASS (227.43 s)

@menli820 It's also including a virtio-blk case, would you like to have a look? thanks~

File: qemu/tests/cfg/block_iscsi_format_large_size_disk.cfg [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    emulated_image_size = ${image_size_stg1}
+    clean_cmd="dd if=/dev/zero of=%s count=256 bs=1M oflag=direct && sync && partprobe"
+    cmd_set_max_sector = "echo 64 > /sys/block/{}/queue/max_sectors_kb"
+    cmd_get_max_segment = cat /sys/block/{}/queue/max_segments
+    file_size = 2147483648
+    guest_cmd = "WIN_UTILS:\coreutils\DummyCMD.exe {} ${file_size} 1"
+    Windows:
+      labeltype = msdos
+      fstype = ntfs
+    Linux:
```
Comment (menli820) [inline qemu/tests/cfg/block_iscsi_format_large_size_disk.cfg:27]: This seems meaningless if above limit   ‘only Windows ‘ on line4

Comment (qingwangrh) [reply qemu/tests/cfg/block_iscsi_format_large_size_disk.cfg]: updated.

**Comment by menli820** [comment]
LGTM.

 (1/3) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_virtio.q35: PASS (147.78 s)
 (2/3) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_hd.q35: PASS (138.69 s)
 (3/3) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.block_iscsi_format_large_size_disk.with_scsi_block.q35: PASS (138.06 s)

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**